modules/shortcodes/dailymotion.php

```diff
@@ -327,3 +327,61 @@ function jetpack_dailymotion_embed_reversal( $content ) {
 }
 
 add_filter( 'pre_kses', 'jetpack_dailymotion_embed_reversal' );
+
+/**
+ * DailyMotion Widget Shortcode
+ *
+ * Example:
+ * [dailymotion-widget id=XYZ]
+ *
+ * @param array $atts
+ * @return string html
+ */
+function jetpack_dailymotion_widget_shortcode( $atts ) {
+	if ( isset( $atts['id'] ) ) {
+		$id = $atts['id'];
+	} else {
+		return '<!--Dailymotion error: bad or missing ID-->';
+	}
+
+	$output="<div class='dailymotion-widget' data-placement='" . $id . "'></div>";
+	$output .= '<script>(function(w,d,s,u,n,e,c){w.PXLObject = n; w[n] = w[n] || function(){(w[n].q = w[n].q || []).push(arguments);};w[n].l = 1 * new Date();e = d.createElement(s); e.async = 1; e.src = u;c = d.getElementsByTagName(s)[0]; c.parentNode.insertBefore(e,c);})(window, document, "script", "//api.dmcdn.net/pxl/client.js", "pxl");</script>';
+
+	return $output;
+}
+
+add_shortcode( 'dailymotion-widget', 'jetpack_dailymotion_widget_shortcode' );
+
+/**
+ * Embed Reversal for Dailymotion widget
+ */
+function jetpack_dailymotion_widget_reversal( $content ) {
+
+	if ( false === stripos( $content, 'dailymotion-widget' ) ) {
```

Comment:
@jeherve  are you sure this is what we wanted to do? I'm not looking for the [dailymotion-widget] shortcode but for the simple "dailymotion-widget" string (that appears in `<div class="dailymotion-widget" data-placement="XXXXXXXXX">`). 

I thought that this was the way to do as this is what is done in other embed reversals, for instance:
- `function dailymotion_channel_reversal( $content ) {
	if ( false === stripos( $content, 'dailymotion.com/badge/' ) ) {
		return $content;
	}`
- `function dailymotion_embed_to_shortcode( $content ) {
	if ( false === stripos( $content, 'www.dailymotion.com/swf/' ) ) {
		return $content;
	}`